src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/handlers/VideoBookingAmendedEventHandler.kt

```diff
@@ -43,24 +45,35 @@ class VideoBookingAmendedEventHandler(
               }
             }
 
-          val appointmentTypesForPrisoner = (history.appointments().map { it.prisonerNumber to it.appointmentType } + booking.appointments().map { it.prisonerNumber to it.appointmentType }).toSet()
-          appointmentTypesForPrisoner.forEach { (prisonerNumber, type) ->
-            val oldAppointment = history.appointments().singleOrNull { it.appointmentType == type && it.prisonerNumber == prisonerNumber }
-            val newAppointment = booking.appointments().singleOrNull { it.appointmentType == type && it.prisonerNumber == prisonerNumber }
+          if (activitiesService.isAppointmentsRolledOutAt(booking.prisonCode())) {
```

Comment:
At the moment I have not altered the behaviour for A&A prisons, only the non A&A prisons.